src/libs/web3/noconnector.ts

```diff
@@ -0,0 +1,34 @@
+import { Actions, Connector } from '@web3-react/types';
+
+export interface NoConnectorArgs {
+  actions: Actions;
+  name: string;
+  url: string;
+}
+
+export class NoConnectorError extends Error {
+  public constructor(name: string) {
+    super(`${name} not installed, redirecting to website...`);
+    this.name = NoConnectorError.name;
+    Object.setPrototypeOf(this, NoConnectorError.prototype);
```

Comment:
This is the first time I see this kind of implementation. It seems a little hacky.

Reply:
Same as above